plugins/modules/web_infrastructure/nomad_job.py

```diff
@@ -0,0 +1,222 @@
+#!/usr/bin/python
+# coding: utf-8 -*-
+
+# (c) 2020, FERREIRA Christophe <christophe.ferreira@cnaf.fr>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: nomad_job
+author: "FERREIRA Christophe (@chris93111)"
+version_added: "1.2.0"
+short_description: Launch an Nomad Job.
+description:
+    - Launch an Nomad job. See
+      U(https://www.nomadproject.io/api-docs/jobs/) for an overview.
+options:
+    host:
+      description:
+        - FQDN of nomad server.
+      required: true
+      type: ...
+    secure:
+      description:
+        - ssl connection.
+    timeout:
+      description:
+        - timeout of request nomad.
+    validate_certs:
+      description:
+        - skip validation cert.
+    cert:
+      description:
+        - path of certificate ssl.
+    key:
+      description:
+        - path of certificate key ssl.
+    namespace:
+      description:
+        - namespace for nomad.
+    token:
+      description:
+        - acl token for authentification.
+    name:
+      description:
+        - name of job for get|stop.
+    state:
+      description:
+        - type of request
+      choices: ["created", "started", "stopped"]
```

Comment:
You need to document what these states do. Also, Ansible states usually contain `present` and `absent`. Please look at state names used by other modules to decide what makes sense here.

Reply:
OK it's apply